forge/Developer Guide/tao-on-mac.md

```diff
@@ -0,0 +1,263 @@
+<!--
+parent: 'Developer Guide'
+created_at: '2010-02-12 11:00:00'
+updated_at: '2010-02-12 11:00:00'
+authors:
+    - 'Bartlomiej Marszal'
+tags:
+    - 'Onboarding'
+    - 'Developer Guide'
```

Comment:
Onboarding is not a useful tag, it would create this element in the Hub navigation. I for instance use Macs only sporadically and would also benefit from your article. Please read https://hub.taotesting.com/taohubdocs/tags-and-tag-groups#tags-in-groups and use something like 'Installation:TAO on MacOS' instead